master/getting-started/kubernetes/index.md

```diff
@@ -42,6 +42,12 @@ the host. Instead, continue directly to the
    sudo apt-get update && sudo apt-get upgrade
    ```
    
+1. Restart Docker.
+
+   ```
+   sudo systemctl restart docker
```

Comment:
Do we know exactly why is this needed? And are we sure it is needed every time?

Reply:
It clears up failures that can occur after dockerd gets updated. It might not be needed every time, but there's no harm in executing it and executing it means the difference between success and failure in the procedure....so it creates a more robust and reliable experience for the readers. Thanks!

Reply:
IMO putting it in the instructions makes it seem like Calico installation requires restarting the docker daemon, which it does not. I'm strongly against mentioning this at all.

Reply:
Not sure I understand your concern....can we find some time to discuss more in person? Thanks @ozdanborne

Reply:
My personal preference would be to ditch both this and the apt-get update / upgrade lines (which are Ubuntu specific anyway even though this doc supports other distros).

IMO the only excuse to suggest running this kind of thing is if you've added an apt source.

Reply:
The reason I had the apt-get update/upgrade bit is because it appeared in the source. It does add a lot of time to the procedure and other people have commented about this. (Discussions of other distros is something I have on my "to do".) Looking into this more, I see that that main kubeadm install page also has them running this command just prior so it's pretty redundant. I'm OK dropping these two.